### ISSUE TITLE
Use a specific page as starting point for rendering

### ISSUE THREAD

**Issue by lucfranken** [Meteor-Community-Packages/react-router-ssr#6]
While running this package with Meteor METEOR@1.11 and  "react": "^16.13.0" I find that loading the homepage (/) works great. The package loads the pre-rendered page and it's quick.

When I click on a button to go to a page this also goes perfect.

When I am at a specific url (say /accounts/sign-up) and I hit refresh things go wrong: Errors appear that the server side content does not match the client state: 

```
Warning: Expected server HTML to contain a matching <div> in <div>.
    in div (created by Row)
    in Row (created by Register)
    in div (created by Container)
    in Container (created by Register)
    in Register (created by AccountRoutes)
    in Route (created by AccountRoutes)
    in Switch (created by AccountRoutes)
    in AccountRoutes
    in div (created by WebContainer)
    in div (created by WebContainer)
    in WebContainer
    in Route
    in Switch
    in Router
    in Switch (created by ReactRouterSSR)
    in Router (created by ReactRouterSSR)
    in ReactRouterSSR
```

I see that the package is trying to match the homepage instead of the specific page at this url. 

I expect this might be related with this part from the readme: 

> The component that encompasses your application. Can be any React element. Routers and Switches are handled by the package so those aren't necessary inside your app.

The main entry point of this application is a <Router> tag from React Router. Is this a Router in Router problem?

Versions:

```
    "react-helmet": "^6.1.0",
    "react-router": "^5.2.0",
    "react-router-bootstrap": "^0.25.0",
    "react-router-dom": "^5.2.0",
```

**Comment by copleykj** [comment]
So I've tracked this issue down to code execution order issue that exists in the `staringatlights:inject-data` package which is a dep of the `staringatlights:fast-render` package which in turn this package depends on. I'm in the process now of moving these packages under community maintenance, and once that is finished, I will fix the issue in the inject-data package and then make updates to this package.

**Comment by copleykj** [comment]
Just released v3.0.0 which now uses `communitypackages:inject-data` and fixes this issue.
